glue/tests/test_main.py

```diff
@@ -18,3 +37,13 @@ def test_no_duplicate_loading(capsys):
     for acall in info.call_args_list:
         if 'Loading plugin' in acall[0][0]:
             assert 'failed' in acall[0][0]
+
+
+def test_list_plugins():
+    """
+    Regression test for retrieving the list of currently loaded plugins
+    """
+    load_plugins(require_qt_plugins=False)
+    plugins = list_plugins()
+    assert isinstance(plugins, list)
+    assert len(plugins) == 5
```

Comment:
Sorry I didn't get back to you sooner about this - rather than checking the absolute number of plugins, you might want to just check that a specific set of them are inside the returned list (but there could be others present)